server/mergin/sync/private_api.yaml

```diff
@@ -581,6 +594,46 @@ components:
             - owner
             - writer
             - reader
+        invitation:
+          description: Present only for type `invitation`
+          type: object
+          required:
+            - "expires_at"
+          properties:
+            expires_at:
+              type: string
+              format: date-time
+              example: 2018-11-30T08:47:58.636074Z
+            projects:
```

Comment:
Access detail for project contains invitation with information about other projects? What for?